src/main/app/forms/models/multiRowFormItem.ts

```diff
@@ -0,0 +1,12 @@
+export abstract class MultiRowFormItem {
+
+  abstract deserialize (input?: any): MultiRowFormItem
+
+  isEmpty (): boolean {
```

Comment:
Have you tried something like:
```ts
return Object.values(this).every(value => !value)
```
or alternatively:
```ts
return !Object.values(this).some(value => value)
```

Reply:
What is this function supposed to do? Check that inputs are blank?

Reply:
All fields are empty (value is falsy: undefined / '')